docs/book/src/reference/versions.md

```diff
@@ -150,16 +153,20 @@ Each Cluster API contract version defines a set of rules a provider is expected
 Those rules can be in the form of CustomResourceDefinition (CRD) fields and/or expected behaviors to be implemented.
 See [provider contracts](../developer/providers/contracts/overview.md)
 
-Each Cluster API release supports only one contract version, and by convention the supported contract version matches
+Each Cluster API release supports one contract version, and by convention the supported contract version matches
```

Comment:
Should we be slightly more explicit by saying the apiVersion of the core apigroup? (the one that contains Cluster/Machine vs. KCP / CABPK)

Reply:
I don't think it is necessary, we have a table with supported API version right above, and we are keeping versions of in tree providers aligned (I also don't want readers or us to start speculating on what happen if versions are not aligned for now).